tests/bugzilla_test.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+
+import json
+import uuid
+
+import loginsightwebhookdemo
+import loginsightwebhookdemo.bugzilla
```

Comment:
Consider using `requests-mock` to avoid the need to round-trip to a real server.